source/math/FloatBox2D.ooc

```diff
@@ -152,9 +153,11 @@ FloatBox2D: cover {
 		this createAround(newCenter, newSize)
 	}
 	toString: func -> String { "#{this leftTop toString()}, #{this size toString()}" }
-	parse: static func (input: String) -> This {
+	parse: static func (input: Text) -> This {
 		array := input split(',')
```

Comment:
Agreed, especially since it's not even an array anymore (it used to be an `ArrayList`). It can be fixed later, I guess.